pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -408,10 +408,10 @@ func (r *multiCIDRRangeAllocator) occupyCIDRs(node *v1.Node) error {
 					return fmt.Errorf("failed to parse CIDR %s on Node %v: %w", cidr, node.Name, err)
 				}
 
-				klog.Infof("occupy CIDR %s for node: %s", cidr, node.Name)
+				klog.InfoS("occupy CIDR for node", "cidr", cidr, "nodeName", node.Name)
 
 				if err := r.Occupy(clusterCIDR, podCIDR); err != nil {
-					klog.V(3).Infof("Could not occupy cidr: %v, trying next range: %w", node.Spec.PodCIDRs, err)
+					klog.V(3).InfoS("Could not occupy cidr, trying next range", "podCIDRS", node.Spec.PodCIDRs, "err", err)
```

Comment:
Pick one "node.Spec.PodCIDRS" or "podCIDRs" -- please make this consistent across this change -- i.e. this comment applies not just on this line.

Reply:
yeah, `podCIDRs` is ok.